rust/src/polars/mod.rs

```diff
@@ -498,7 +505,8 @@ impl From<LazyFrame> for OnceFrame {
             Ok(match query {
                 Query::External(q_external) => Answer::External(match q_external {
                     OnceFrameQuery::Collect => {
-                        let dataframe = lazyframe.collect()?;
+                        // TODO: allow CSE once https://github.com/pola-rs/polars/pull/23032 is released
```

Comment:
File a tracking issue on our side and reference it here, if we want to be sure not to miss this?